temoa/utilities/graphviz_util.py

```
@@ -0,0 +1,126 @@
+from collections.abc import Callable, Iterable, Sequence, Sized
+
+
+def get_color_config(grey_flag: bool) -> dict[str, str | tuple[str, ...]]:
```

Ruff (0.14.5)

Boolean-typed positional argument in function definition

(FBT001)